### DISCUSSION TITLE
Can I use the offcial github runner images?

### DISCUSSION THREAD

**Discussion by liudonghua123** [nektos/act#1504] [General]
Hi, I read the docs on https://github.com/nektos/act/blob/master/IMAGES.md, it seems the images list there are third-parties, not the offcial ones. Is there any reasons why these images are required or recommend then the offcial ones?

And can I use the offcial github runner images? And how to do it?

**Comment by KnisterPeter** [comment]
You can take this one:
https://github.com/nektos/act#alternative-runner-images

**Reply by liudonghua123** [reply]
@KnisterPeter Thanks, I am not familiar with github runner images, is the official https://github.com/actions/runner-images only provides VM template, not docker images, while  https://github.com/nektos/act#alternative-runner-images use some docker images built from https://github.com/actions/runner-images?

**Reply by KnisterPeter** [reply]
Me neither. Sorry.

**Reply by catthehacker** [reply]
GitHub doesn't provide images at all due to licencing issues
